config.template

```diff
@@ -121,6 +121,18 @@ areaNH = IceArea_timeseries/iceAreaNH_climo.nc
 areaSH = IceArea_timeseries/iceAreaSH_climo.nc
 volNH = PIOMAS/PIOMASvolume_monthly_climo.nc
 volSH = none
+concNASATeamWinNH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_NH_jfm.interp0.5x0.5.nc
+concNASATeamSumNH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_NH_jas.interp0.5x0.5.nc
+concNASATeamWinSH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_SH_djf.interp0.5x0.5.nc
+concNASATeamSumSH = SSMI/NASATeam_NSIDC0051/SSMI_NASATeam_gridded_concentration_SH_jja.interp0.5x0.5.nc
+concBootstrpWinNH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_NH_jfm.interp0.5x0.5.nc
+concBootstrpSumNH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_NH_jas.interp0.5x0.5.nc
+concBootstrpWinSH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_SH_djf.interp0.5x0.5.nc
+concBootstrpSumSH = SSMI/Bootstrap_NSIDC0079/SSMI_Bootstrap_gridded_concentration_SH_jja.interp0.5x0.5.nc
+thickonNH = ICESat/ICESat_gridded_mean_thickness_NH_on.interp0.5x0.5.nc
+thickfmNH = ICESat/ICESat_gridded_mean_thickness_NH_fm.interp0.5x0.5.nc
+thickonSH = ICESat/ICESat_gridded_mean_thickness_SH_on.interp0.5x0.5.nc
+thickfmSH = ICESat/ICESat_gridded_mean_thickness_SH_fm.interp0.5x0.5.nc
 
```

Comment:
Regarding 2, I think we really want to avoid hard coding file names in the analysis scripts.  In general, we want to move toward analysis that is more general and not so targeted at producing a specific plot from a specific type of run.  So I think this is a (small) step in that direction.